index.html

```diff
@@ -1,270 +1,393 @@
-                    <span class="bar"></span>
-                </div>
-            </nav>
-        </header>
-
-    <section class="container hero-wrapper">
-        <div class="hero-container">
-            <h1 class="headline">Hey There.<br>I'm Mehria</h1>
-            <h2 class="job-title">I am a web full stack developer </h2>
-            <p class="job-description">I can help you build a product, feature or website.Look through some of my work and experience! If you like what you see and have a project you need coded, do not hestiate to contact me.</p>
+    <header class="header">
+    
+    
+        <nav class="statusbar">
+            <div class="menu-desktop-container">
+                <a class="menu-desktop" href="#">Portoflio</a>
+                <a class="menu-desktop" href="#">About</a>
+                <a class="menu-desktop" href="#">Contact</a>
+                <a class="menu-desktop" href="#"><img alt="Email"
+                        src="assets/mail.png"></a>
+            </div>
+            <div class="mobile">
+                <h1 class="logo2">Mehria</h1>
+                <img class="humb" src="./assets/Union.png" alt="">
+              </div>
+        </nav>
+    </header>
+    <section class="page1-content blackpage">
+        <div class="header-text">
+            <h1 class="my-name">Hey There.<br>I'm Mehria</h1>
+            <h2 class="my-position">I’am a Software Developer</h2>
+            <p class="discription-me">I am a software developer a graphic designer and a video journalist.
+                I can help you build a product, feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need to be coded,
+                don’t hesitate to contact me.
+            </p>
         </div>
-        
-        <div class="links d-flex">
-            <ul>
-                <li>
-                    <a href="https://github.com/Mehria-Rahmani/Hello-Microverse.git" id="git" target="blank"><img src="assets/github.svg" alt="github"></a>
-                </li>
-                <li>
-                    <a href="https://www.linkedin.com/in/mehria-rahmani-379211222" id="linkedin" target="blank"><img src="assets/linkedin.svg" alt="linkedin"></a>
-                </li>
+        <div class="page-footer">
+            <div class="social-media">
+                <ul class="icon-list">
+                    <li class="img-list">
+                        <a  href="https://github.com/Mehria-Rahmani/Hello-Microverse.git" target="_blank">
+                            <img class="github icons" alt="Github Icon" src="assets/github.svg">
+                        </a>
+                    </li>
+                    <li class="img-list">
+                        <a href="https://www.linkedin.com/in/mehria-rahmani-379211222" target="_blank">
+                            <img class="linkedin icons" alt="LinkedIn Icon" src="assets/Linkedin.svg">
```

Comment:
- [ ] [OPTIONAL] The LinkedIn icon is broken 👇 
![Screenshot from 2022-06-11 13-50-49](https://user-images.githubusercontent.com/87965534/173184752-4d8b4694-99e8-4010-aba0-eba24f2ecac9.png)
Please fix the icon name, you should use `src="assets/linkedin.svg"` instead of `src="assets/Linkedin.svg"`